### PR TITLE
[FEATURE] PSR-4 Compliance LRN-33188

### PR DESCRIPTION
**Pull request by rhiannon-eldridge-lrn**
## Checklist

- [X] Feature
- [ ] Bug

- [X] ChangeLog.md updated

- [ ] Tests added
- [X] All testsuite passes
- [X] `make dist` completed successfully

### CODE REVIEW

**Review by TheRealEdDawson — APPROVED** [review]
Looks great. Love the one-liner to start up the web server. Thanks Rhi.

**Review by aaron-ji-lrn — COMMENTED** [review]
looks good, will do func

File: Makefile [hunk 1]
```diff
@@ -53,21 +53,19 @@ build-clean: clean
 # build a dist zip file from the distdir, THEN run the tests in the dist dir,
 # to avoid polluting the distfile with dev dependencies
 ###
-dist: dist-zip dist-test
+dist: dist-test
```
Comment (aaron-ji-lrn) [inline Makefile:56]: Why do we remove `dist-zip`?  according to the comments above, we need to build a dist zip.

Comment (rhiannon-eldridge-lrn) [reply Makefile:56]: `dist-test` also calls `dist-zip`, so calling both is a bit redundant.

File: Makefile [hunk 2]
```diff
@@ -56,11 +54,13 @@ build-clean: clean
 dist: dist-test
```
Comment (aaron-ji-lrn) [inline Makefile:54]: There are two rules point to `dist-test`, 
```
test-dist: dist-test
dist: dist-test
```

I feel they are duplicated, if we just keep one will be more concise.

**Review by aaron-ji-lrn — APPROVED** [review]
good to go

**Pull request merged**